2.0/m_ascii.cpp

```diff
@@ -0,0 +1,55 @@
+/*
+ * InspIRCd -- Internet Relay Chat Daemon
+ *
+ *   Copyright (C) 2016 Adam <Adam@anope.org>
+ *
+ * This file is part of InspIRCd.  InspIRCd is free software: you can
+ * redistribute it and/or modify it under the terms of the GNU General Public
+ * License as published by the Free Software Foundation, version 2.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "inspircd.h"
+
+/* $ModAuthor: Adam */
+/* $ModAuthorMail: Adam@anope.org */
+/* $ModDesc: Implements CASEMAPPING=ascii */
+/* $ModDepends: core 2.0 */
+/* $ModConflicts: m_nationalchars.so */
+
+class ModuleASCII : public Module
+{
+ public:
+	void init()
+	{
+		national_case_insensitive_map = ascii_case_insensitive_map;
```

Comment:
This might have been replaced by another module. You should probably store the original value in a private variable and reset it to that in the dtor.